lib/storage/bucket.js

```diff
@@ -257,6 +272,11 @@ Bucket.prototype.getFiles = function(query, callback) {
     callback = query;
     query = {};
   }
+  if (query.folder) {
+    query.delimiter = '/';
+    query.prefix = query.folder + '/';
```

Comment:
Nope, it's very dumb. This is largely a POC so not sure how we should handle those cases.